sqlite/src/vdbeaux.c

```diff
@@ -5758,6 +5758,9 @@ Mem *sqlite3CloneResult(
     if( !pMem ) return 0;
   }else{
     *pSize -= memRowSize(pMem, nCols);
+    for(i=0; i<nCols; ++i){
+      sqlite3_value_free_inplace(pMem + i);
```

Comment:
For  consistency with the sqlite3CloneResultFree function, maybe this?

sqlite3_value_free_inplace(&pMem[i]);

Reply:
Okay